index.bs

```diff
@@ -956,8 +961,8 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 
 1. Let |issuedRequests| be a new [=ordered set=].
 
-1. Let |authenticators| represent a value which at any given instant is a [=set=] of platform-specific handles, where each
-    [=set/item=] identifies an [=authenticator=] presently available on this platform at that instant.
+1. Let |authenticators| represent a value which at any given instant is a [=set=] of [=client device=]-specific handles, where each
```

Comment:
[=client device=] -> [=client platform=]      ?

Reply:
Yeah, I considered that too. Done.